Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM alpine:3.5
+
+RUN apk --update add coreutils
+
+WORKDIR /shredder/
+COPY bin/shredder /shredder/bin/shredder
+
+ENTRYPOINT ["/shredder/bin/shredder"]
```

Comment:
I usually agree. For an image like this that's meant to be invoked almost like a regular bin, though, I think it makes usage a bit nicer (even if less common actions like "give me a shell" are a bit more of a pain).